releng/org.eclipse.xtext.contributor/Xtext.setup

```diff
@@ -884,7 +887,40 @@
           <repository
               url="${p2.draw2d}"/>
           <repository
-              url="http://download.eclipse.org/eclipse/updates/4.9-I-builds"/>
+              url="http://download.eclipse.org/eclipse/updates/4.9-I"/>
```

Comment:
remove

Reply:
The additional eclipse/updates repository is only needed when the base platform target should be overridden. The Eclipse Platform already defines the typical main repos for each release. Only for the upcoming release we need the I-build repo additionally.